src/components/dropdowns/CynDropdown.vue

```diff
@@ -1,40 +1,34 @@
 <template>
-    <div id="dropdown-div">
-        <label for="users">Select Users:</label>
-        <br />
-        <select v-model="selected">
-            <option value disabled>Please Select a User</option>
-            <option v-bind:value="user.name" v-for="user in users" :key="user.id"
-                >{{ user.name }} | ID: {{ user.id }}</option
-            >
-        </select>
-        <p>Current user selected: {{ selected }}</p>
-    </div>
+  <div id="dropdown-div">
+    <label for="users">Select an Item:</label>
+    <br />
+    <select @change="onChange($event)">
+      <option value disabled selected>Please Select an item</option>
+      <option
+        v-for="item in items"
+        :value="item.name"
+        :key="item.id"
+      >{{ item.name }} | ID: {{ item.id }}</option>
+    </select>
+    <p>Current user selected: {{ selectedItem }}</p>
+  </div>
 </template>
 
 <script>
-import axios from 'axios';
-
 export default {
-    name: 'CynDropdown',
-    data() {
-        return {
-            users: null,
-            selected: 'None',
-            defaultSelected: 'Please select a user'
-        };
-    },
-    created() {
-        axios
-            .get('https://jsonplaceholder.typicode.com/users')
-            .then(res => {
-                this.users = res.data;
-            })
-            .catch(err => {
-                this.users = 'An error has occured: ' + err;
-            });
+  name: "CynDropdown",
+  props: {
+    items: Array,
+    selectedItem: String
```

Comment:
I know we went over this a bit already on slack as well, so just for reference here..

We are correctly passing in `items` as a prop here, because it lets the person using the component decide what goes into the dropdown list. But `selectedItem` is not something we want the developer to be choosing when they are implementing the component, it is determined by the current `state` of the dropdown component - what item the user clicks on.
So, we are going to want to declare `selectedItem` in the `computed` block, and then set the value when the user selects an option with a simple `v-on:click` helper.